src/assets/scss/_buttons.scss

```diff
@@ -2,12 +2,12 @@
 //
 // Custom buttons for the docs.
 
+// scss-docs-start btn-css-vars-example
```

Comment:
This line and `// scss-docs-end btn-css-vars-example` are not needed in this repo since we don't try to display this content in the documentation like it's done upstream. If it's for maintenance purpose (copy and paste upstream file), let's keep it.